app/lib/features/settings/widgets/in_settings.dart

```diff
@@ -9,14 +9,13 @@ class InSettings extends StatelessWidget {
   Widget build(BuildContext context) {
     return LayoutBuilder(
       builder: (context, constrains) {
-        if (constrains.maxWidth > 700) {
+        if (constrains.maxWidth > 770) {
           return Row(
+            mainAxisSize: MainAxisSize.min,
+            mainAxisAlignment: MainAxisAlignment.start,
             children: [
-              const SizedBox(
-                width: 350,
-                child: SettingsMenu(),
-              ),
-              Expanded(child: child),
+              const Flexible(flex: 1, child: SettingsMenu()),
```

Comment:
are we actually certain, this works on width == 771? Because 771/3 = 257, which is 100 smaller than the 350 it had before ...